src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json

```diff
@@ -0,0 +1,2021 @@
+﻿{
+  "Description": "Sysbench Database Server Performance Workload",
+  "MinimumExecutionInterval": "00:01:00",
+  "Parameters": {
+    "DatabaseName": "sbtest"
+  },
+  "Actions": [
+    {
+      "Type": "SysbenchOLTPServerExecutor",
+      "Parameters": {
+        "Scenario": "SysbenchOLTPServer",
+        "Role": "Server"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_read_write_T2_TB16_REC1000",
```

Comment:
We might have too many combinations here. How long would a say 16 core machine take to finish everything once?

Reply:
i think the 16-core-specific scenarios might finish these in 3.5 hours but i can definitely cut some